kirin.proto

```diff
@@ -5,6 +5,7 @@ import "gtfs-realtime.proto";
 
 extend transit_realtime.TripUpdate {
     optional string trip_message = 1000;
+    optional string company = 1001;
```

Comment:
To me, it would fit better un `TripDescriptor`, named `company_id` (unless we pass something else but I guess we're gonna pass id) : https://developers.google.com/transit/gtfs-realtime/reference/#message_tripdescriptor

Reply:
@pbougue I moved company in TripDescriptor.